timelock-agent/src/main/java/com/palantir/timelock/config/TimeLockRuntimeConfiguration.java

```diff
@@ -44,6 +49,18 @@ public long slowLockLogTriggerMillis() {
         return 10000;
     }
 
+    @JsonProperty("auth-clients")
+    @Value.Default
+    public List<TimelockAuthClient> authClients() {
+        return ImmutableList.of();
+    }
+
+    @JsonProperty("admin-token")
```

Comment:
what can I do with an admin-token? and why is an admin token special-snowflaked?